cdap-ui/app/cdap/components/NamespaceDropdown/index.js

```diff
@@ -240,7 +241,9 @@ export default class NamespaceDropdown extends Component {
               <small>{T.translate('features.Navbar.NamespaceDropdown.namespaceLabel')}</small>
               <span>{currentNamespace}</span>
             </div>
-            <span className="fa fa-caret-down float-xs-right" />
+            <span className="float-xs-right">
+              <SVG icon="icon-caret-down" />
```

Comment:
I think it gives more flexibility. Before when we were still using `<i>` font icons, at times we would give the `<i>` a class, while still wrapping it inside another `<span>`.